.github/workflows/issue-view.yml

```diff
@@ -0,0 +1,384 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
nit: can you please change this comment. And this one is for the detailed list of issues/Prs , right?

Reply:
ok pr is separate

Reply:
removed the comment